### ISSUE TITLE
Add nowcasting to the historical chart

### ISSUE THREAD

**Issue by hnykda** [epidemics/covid#367]
blocked by #251

Nowcasting = forecast for next two weeks based mainly on exponential regression using the estimate of growth rates

AC:
* the historical graph contains now casting

**Comment by hnykda** [comment]
Also blocked by #449

**Comment by jk3141** [comment]
Likely should be superseded by adding the results of countermeasures model to the website.

**Comment by hazarane** [comment]
replaced by #477
